wpilibj/src/test/java/edu/wpi/first/wpilibj/hal/MatchInfoDataTest.java

```diff
@@ -9,13 +9,79 @@
 
 import org.junit.jupiter.api.Test;
 
+import edu.wpi.first.hal.sim.mockdata.DriverStationDataJNI;
+import edu.wpi.first.wpilibj.DriverStation.MatchType;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
 class MatchInfoDataTest {
   @Test
-  @SuppressWarnings("PMD.JUnitTestsShouldIncludeAssert")
-  void matchInfoDataDoesNotThrow() {
+  void matchInfoDataEventName() {
+    HAL.initialize(500, 0);
```

Comment:
Sorry, one more thing.  You do not need the HAL.initalize.  JUnit calls it for you.